vncviewer/CMakeLists.txt

```diff
@@ -100,8 +100,20 @@ if(UNIX)
       DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/vncviewer.desktop.in
     )
   endif()
-  add_custom_target(desktop ALL DEPENDS vncviewer.desktop)
-  install(FILES ${CMAKE_CURRENT_BINARY_DIR}/vncviewer.desktop DESTINATION ${CMAKE_INSTALL_FULL_DATADIR}/applications)
+  # FIXME: Add translations to org.tigervnc.vncviewer.metainfo.xml
+  add_custom_command(OUTPUT org.tigervnc.vncviewer.metainfo.xml
+    COMMAND cp ${CMAKE_CURRENT_SOURCE_DIR}/org.tigervnc.vncviewer.metainfo.xml.in org.tigervnc.vncviewer.metainfo.xml
+    DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/org.tigervnc.vncviewer.metainfo.xml.in
+  )
+
+  add_custom_target(desktop ALL
```

Comment:
Perhaps prudent to rename this, or split things to two targets. This name gets a bit confusing otherwise.

Reply:
Has been split.